src/elements-experimental/pearl-chain-vertical/pearl-chain-vertical.visual.spec.ts

```diff
@@ -0,0 +1,297 @@
+import {
+  describeViewports,
+  visualDiffDefault,
+} from '@sbb-esta/lyne-elements/core/testing/private.js';
+import '@sbb-esta/lyne-elements/icon.js';
+import { html } from 'lit';
+
+import './pearl-chain-vertical.js';
+import '../pearl-chain-vertical-item.js';
+
+describe(`sbb-pearl-chain-vertical`, () => {
+  const defaultArgs = {
+    lineType: 'standard',
+    lineColor: 'default',
+    bulletType: 'default',
+    minHeight: '100',
+    hideLine: false,
+    bulletSize: 'start-end',
+    position: 0,
+  };
+
+  describeViewports({ viewports: ['medium'] }, () => {
+    it(
+      'connection',
+      visualDiffDefault.with(async (setup) => {
+        const args = {
+          ...defaultArgs,
+          lineColor: 'disruption',
+          hideLine: true,
+          bulletSize: 'stop',
+        };
+        await setup.withFixture(html`
+          <sbb-pearl-chain-vertical>
+            <sbb-pearl-chain-vertical-item .pearlChainVerticalItemAttributes=${defaultArgs}>
+              <div
+                slot="right"
+                style="--sbb-pearl-chain-vertical-right-item-block-start: -8px; --sbb-pearl-chain-vertical-right-item-inline-start: 10px;"
+              >
+                <div style="display: flex; flex-direction: row; justify-content: space-between;">
+                  <div>Station</div>
+                  <div>Pl. 12</div>
+                </div>
+                <div style="display: flex; flex-direction: row; justify-content: space-between;">
+                  <div>
+                    <sbb-icon role="img" name="train-small" aria-hidden="true"></sbb-icon>
+                    <sbb-icon role="img" name="ir-27" aria-hidden="true"></sbb-icon>
+                    <div>Direction Station</div>
+                  </div>
+                  <span>
+                    1.<sbb-icon name="utilization-high"></sbb-icon> 2.<sbb-icon
+                      name="utilization-high"
+                    ></sbb-icon>
```

Comment:
shouldn't here the timetable-occupancy be used? as well in stories probably. (valid also for following examples)

Reply:
On sbb.ch they use the timetable-occupancy, I changed it in both visual and stories